src/components/Explore/LocationSelector.tsx

```diff
@@ -1,18 +1,32 @@
 import React, { useState } from 'react';
 import Grid from '@material-ui/core/Grid';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
-import { Button } from '@material-ui/core';
 import { useTheme } from '@material-ui/core/styles';
 import { Modal } from '@material-ui/core';
 import { Location } from 'common/locations';
 import * as Styles from './LocationSelector.style';
 import AutocompleteLocations from 'components/AutocompleteLocations';
 
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import * as ExploreStyles from './Explore.style';
+
 const LocationSelector: React.FC<{
   locations: Location[];
   selectedLocations: Location[];
   onChangeSelectedLocations: (newLocations: Location[]) => void;
-}> = ({ locations, selectedLocations, onChangeSelectedLocations }) => {
+  hasMultipleLocations: boolean;
```

Comment:
Rather than passing this into the component separately, couldn't we just do `const hasMultipleLocations = selectedLocations.length >= 1` or something?

Reply:
yep!